core/test/integration/model/model_posts_spec.js

```diff
@@ -180,7 +179,7 @@ describe('Post Model', function () {
                         }).catch(done);
                 });
 
-                it('returns computed fields when columns are asked for explicitly', function (done) {
+                xit('returns computed fields when columns are asked for explicitly', function (done) {
```

Comment:
I think these tests are testing the wrong thing in the context of this change. We move the url generation from the model, so url's should be tested on api layer tests. And from my research, the are covered there, so can be removed from here completely.

Reply:
Atm, adding coverage for this set of changes on unit test level.

Reply:
Depends if all cases are covered either by unit test or routing test. Are they?

Reply:
Discussed: move to unit or routing depends on what they do. Double check if the cases are covered already.